### Introduction ###

The following notebook deals with the tasks assigned throughout the semester

### Libraries/Imports ###

The following python libraries have been imported for use in satisdying the requirements of the assinged tasks.


In [16]:
import math                 #import the math library

---

### Task 1: Permutations and Combinations ###



In [17]:
n = 12                      #number of cups of tea in experiment
k = 6                       #number of cups wiht milk in first
print(math.comb(n,k))       #calculate and print the numner of potential combinations of cups with milk first based on 12 cups without order (order doesnt matter)


924


The above illustrates that the lady has 1 in 924 chances of correctly grouping the cups of tea assuming she has no special ability identify cups of tea when there are 12 cups of tea, 6 of whcih had milk put in first.

In [18]:
#calculate pobability of the lady correctly selecting the 6 correct cups.

print("Probability of Success: " + str(1/ math.comb(n,k)))

Probability of Success: 0.0010822510822510823


---

### Task 2: numpy's Normal Distribution ###

---

### Task 3: t-Test Calculation ###

---

### Task 4: ANOVA ###

---